test/test-util.js

```diff
@@ -60,4 +63,32 @@ describe('extractZip()', () => {
 
 		throw new Error('Expected error');
 	});
+
+	it('should support symlinks', async () => {
+		const tempDir = tmp.dirSync().name;
+		await extractZip({ dest: tempDir, file: path.join(__dirname, 'fixtures', 'symlinks.zip') });
+		console.log(fs.readdirSync(path.join(tempDir, 'symlinks')));
+		const folder = path.join(tempDir, 'symlinks/folder');
+		expect(fs.existsSync(folder)).to.equal(true);
+		const folderStat = fs.statSync(folder);
+		expect(folderStat.isDirectory()).to.equal(true);
+
+		const file = path.join(tempDir, 'symlinks/folder/testfile.txt');
+		expect(fs.existsSync(file)).to.equal(true);
+		const fileStat = fs.statSync(file);
+		expect(fileStat.isDirectory()).to.equal(false);
+		expect(fileStat.isFile()).to.equal(true);
+
+		const fileLink = path.join(tempDir, 'symlinks/link.txt');
+		expect(fs.existsSync(fileLink)).to.equal(true);
+		const fileLinkStat = fs.lstatSync(fileLink);
+		expect(fileLinkStat.isSymbolicLink()).to.equal(true);
+
+		const folderLink = path.join(tempDir, 'symlinks/folderlink');
+		expect(fs.existsSync(folderLink)).to.equal(true);
```

Comment:
This is continuing to fail on the Windows nodes, which is very, very odd. The file listing shows it, so it does "exist", but from some Googling this can return false if the target folder *doesn't* exist or the symlink is broken. It'd be good to actually unzip this on a Windows machine ourselves and see what's up - maybe it's just some weird node issues and we need to skip this on windows tests. Or maybe it's pointing to a real issue?